nginx-ldapauth-proxy/values.yaml

```diff
@@ -1,59 +1,120 @@
 # Default values for nginx-ldapauth-proxy.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
 replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
 image:
-  repository: camptocamp/nginx-auth-ldap
-  tag: 1.17.1-on-alpine-3.5
+  repository: nginx
+  # This sets the pull policy for images.
   pullPolicy: IfNotPresent
-  # pullSecrets:
-   # - docker-secret
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# This is for the secretes for pulling an image from a private repository more information can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
+# This is to override the chart name.
+nameOverride: ""
+fullnameOverride: ""
+
+#This section builds out the service account more information can be found here: https://kubernetes.io/docs/concepts/security/service-accounts/
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# This is for setting Kubernetes Annotations to a Pod.
+# For more information checkout: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ 
+podAnnotations: {}
+# This is for setting Kubernetes Labels to a Pod.
+# For more information checkout: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+# This is for setting up a service more information can be found here: https://kubernetes.io/docs/concepts/services-networking/service/
 service:
-  name: nginx-ldapauth
+  # This sets the service type more information can be found here: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
   type: ClusterIP
-  externalPort: 443
-  internalPort: 80
-proxy:
-  protocol: "ldap"
-  port: 443
-  host: "kubernetes.default.svc.cluster.local"
-  authName: "Auth Required"
-  ldapHost: ""
-  ldapPort: 389
-  ldapGroup: "memberUid"
-  ldapDN: "dc=example,dc=com"
-  ldapFilter: "objectClass=organizationalPerson"
-  ldapBindDN: "cn=auth,dc=example,dc=com"
-  requires:
-    - name: "authGroup"
-      filter: "cn=secret,ou=groups,dc=example,dc=com"
```

Comment:
You are removing all this from the values.yaml file, but the template `nginx-ldapauth-proxy/templates/configmap.yaml` is still using them.

It is optional but enabled by default, so probably you should keep these values and document them.